src/router/index.js

```diff
@@ -55,4 +84,30 @@ const router = new VueRouter({
   routes
 })
 
+router.beforeEach((to, from, next) => {
```

Comment:
This logic should be in the `authMiddleware.js` file

Reply:
Here is an example of how  you can do that
![image](https://user-images.githubusercontent.com/9899327/130565513-7a94b3ac-63fb-42a4-aaa0-50ff03927016.png)